### PR TITLE
Add warning message if you're *ALLOBJ

### PR DESCRIPTION
**Pull request by ThePrez**
### Changes

People shouldn't do development with *ALLOBJ-type authority

### Checklist

* [x] have tested my change
* [x] updated relevant documentation **(n/a)**
* [x] Remove any/all `console.log`s I added
* [x] eslint is not complaining
* [x] **for feature PRs**: PR only includes one feature enhancement.

### CODE REVIEW

**Review by chrjorgensen — CHANGES_REQUESTED** [review]
@ThePrez Nice addition - even though I will be met by this message everytime I connect, since I'm the admin... 😆 (but this may make me change my setup and routines 😃)

Could you add a progress message for this and put the code after it? Since this has nothing to do with temp library config...

File: src/api/IBMi.ts [hunk 1]
```diff
@@ -301,6 +301,12 @@ export default class IBMi {
           message: `Checking temporary library configuration.`
         });
 
+        // An *ALLOBJ user doing development? Naughty! 
+        const allObjCheckResult = await this.sendCommand({command: `test -w /QOpenSys/QIBM/ProdData`, directory: `.` });
+        if (0 == allObjCheckResult.code) {
```
Comment (worksofliam) [inline src/api/IBMi.ts:306]: Please use a strict comparison (`===`)

**Review by worksofliam — CHANGES_REQUESTED** [review]

File: src/api/IBMi.ts (hunk 1 above)
Comment (ThePrez) [reply src/api/IBMi.ts]: Resolved at https://github.com/halcyon-tech/vscode-ibmi/pull/1479/commits/e6a1a5c74850ed2670a2be7d16656a5776af8772

**Comment by ThePrez** [comment]
Tweaked surrounding logic at https://github.com/halcyon-tech/vscode-ibmi/pull/1479/commits/e6a1a5c74850ed2670a2be7d16656a5776af8772

**Review by chrjorgensen — APPROVED** [review]
@ThePrez Approved and merged - thanks!

**Pull request merged**